src/terms/polynomials.h

```diff
@@ -428,5 +428,9 @@ extern bool polynomial_is_const_plus_var(polynomial_t *p, int32_t x);
 extern bool polynomial_is_var(polynomial_t *p, int32_t x);
 
 
+/*
+ * Check if all coefficients are integer and less than mod
+ */
+extern bool polynomial_is_ff_poly(const polynomial_t *p, const rational_t *mod);
```

Comment:
consider renaming this function. 
The name is not telling what it is supposed to do.

Reply:
Good point. What about `polynomial_is_integer_mod`?